package.json

```diff
@@ -162,7 +163,8 @@
   "prettier": "@shopify/prettier-config",
   "stylelint": {
     "extends": [
-      "@shopify/stylelint-plugin/prettier"
+      "@shopify/stylelint-plugin/prettier",
```

Comment:
It might be worth moving our stylelint config our from a key in package.json, into a .styleintrc.js file, and then we can inline the contents of `stylelint/config/index.js into it`